### PR TITLE
Adds appsync log forwarding

### PR DESCRIPTION
**Pull request by IvanTopolcic**
### What does this PR do?

Adds Appsync logging to the logs forwarder. Appsync logs pipeline configurations have been set up already.

### Motivation

New feature request to have better integration with AWS services.

### Testing Guidelines

Unit tests, as well as hand-tested in sandbox.

### Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [x] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [x] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [x] This PR passes the integration tests (ask a Datadog member to run the tests)
- [x] This PR passes the unit tests 
- [x] This PR passes the installation tests (ask a Datadog member to run the tests)

### CODE REVIEW

File: aws/logs_monitoring/parsing.py [hunk 1]
```diff
@@ -291,6 +291,9 @@ def find_cloudwatch_source(log_group):
     if log_group.startswith("/aws/fsx/windows"):
         return "aws.fsx"
 
+    if log_group.startswith("/aws/appsync/"):
+        return "appsync"
```
Comment (nhinsch) [inline aws/logs_monitoring/parsing.py:295]: I think this could be a good opportunity to make all of these sources into constants.

**Review by nhinsch — APPROVED** [review]
Looks good, one suggestion

File: aws/logs_monitoring/parsing.py (hunk 1 above)
Comment (IvanTopolcic) [reply aws/logs_monitoring/parsing.py:295]: Agreed. I'm going to just try and get this feature rolled out for now and I think we can probably consolidate this during engineering week.

Comment (GAVID1020examplysabah) [reply aws/logs_monitoring/parsing.py:295]: من مشکلی ندارم و ضرب‌المثلی هست که میگوید .من ریش و قیچی را به git hub سپرده‌ام .فقط میخواهم بدانم که آیا اینها متعلق به من بوده که شخص هکر آنها را هک کرده .

**Pull request merged**